src/App.tsx

```diff
@@ -7,13 +7,23 @@ import { SnackbarProvider } from "@hooks/useSnackbar";
 import { ConfirmDialogProvider } from "@hooks/useConfirmDialog";
 import { ENVVariable } from "@utils/constants";
 import { Suspense } from "react";
+import { useTranslation } from "react-i18next";
+import { SELECTED_LANGUAGE_KEY, PRIMARY_LANGUAGE } from "./utils/constants";
```

Comment:
At  line 8 of this document you have similar import, please merge it with this one:
`import { ENVVariable } from "@utils/constants";`

Reply:
fixed